environments.json

```diff
@@ -144,11 +144,11 @@
     "family": "Chakra",
     "short": "IE 7",
     "release": "2006-10-18",
-    "obsolete": true,
+    "retire": "2016-01-12",
+    "obsolete": "very",
     "test_suites": [
       "es5",
-      "es6",
-      "esnext",
+      "esintl",
```

Comment:
I'd say that the oldest version in a browser line that should be displayed is the newest one that has zero support - so if IE 8 also has zero support and is visible, then it makes sense to hide IE 7.